src/applications/disability-benefits/all-claims/tests/pages/incidentLocation.unit.spec.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { mount } from 'enzyme';
+
+import { ERR_MSG_CSS_CLASS } from '../../constants';
+
+import { DefinitionTester } from '../../../../../platform/testing/unit/schemaform-utils.jsx';
+import { formConfig781 } from '../../config/781';
+
+describe('PTSD Incident location', () => {
+  const page = formConfig781(1).incidentLocation0;
```

Comment:
I'm wondering now if we should test at least 2 of each page? Or if that doesn't really give us anything. The correct thing would probably be to test the `formConfig781` itself(outputs X amount of config pages etc...).